static/css/styles.css

```diff
@@ -366,6 +366,20 @@ h4 {
     margin-bottom: 0.5em;
 }
 
+h5 {

```

Comment:
We shouldn't be overriding the defaults, especially when using something like Bootstrap. It's designed to provide the correct sizing for most supported browsers and viewports. No need to add all of these styles for the various `<h>` elements. Please consider removing.

Reply:
Removing the h5 & h6 which I believe I added.

Are there anymore or can we resolve this specific convo?